bigbone/src/test/kotlin/social/bigbone/api/method/admin/AdminIpBlocksMethodsTest.kt

```diff
@@ -0,0 +1,35 @@
+package social.bigbone.api.method.admin
+
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+import social.bigbone.api.Range
+import social.bigbone.testtool.MockClient
+
+class AdminIpBlocksMethodsTest {
```

Comment:
As I wrote in the other file: I don’t think we should do client-side validation of the IP address. But if we _did_, the validation should be tested here!

We should also try to test all methods, even if it’s just to ensure that the correct `MastodonClient` method is called with the correct parameters (endpoint, query parameters, method (get, post, …))